source/optional_features/welsh_language/index.html.md.erb

```diff
@@ -19,10 +22,7 @@ tool to add a Welsh service name to your service.
 If you add a Welsh service name to your service, users will see this on Welsh-language
 payment pages instead of the English service name.
```

Comment:
If you’re creating a Welsh payment link, you need to already have a Welsh service name set beforehand to avoid the URL containing the English service name (if you add a Welsh service name later, it won’t change the URLs of existing links). I believe we’ve had contact from services that were caught out by this, so it may be worth highlighting.

Reply:
yep, definitely a good call

Reply:
Good plan! I'll have a rejig.

Reply:
See my comment below - Tracy's going to add this to the admin tool or features page, so the content is closer to where users need it.